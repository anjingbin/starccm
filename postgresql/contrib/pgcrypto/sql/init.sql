--
-- init pgcrypto
--

\set ECHO none
\i pgcrypto.sql
\set ECHO all

-- check for encoding fn's
select encode('foo', 'hex');
select decode('666f6f', 'hex');

