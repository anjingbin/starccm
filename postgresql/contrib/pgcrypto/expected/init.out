--
-- init pgcrypto
--
\set ECHO none
-- check for encoding fn's
select encode('foo', 'hex');
 encode 
--------
 666f6f
(1 row)

select decode('666f6f', 'hex');
 decode 
--------
 foo
(1 row)

